flink-runtime/src/main/java/org/apache/flink/runtime/io/network/metrics/NettyShuffleMetricFactory.java

```diff
@@ -66,6 +66,8 @@
 
 	private static final String METRIC_INPUT_QUEUE_LENGTH = "inputQueueLength";
 	private static final String METRIC_INPUT_POOL_USAGE = "inPoolUsage";
+	private static final String METRIC_INPUT_FLOATING_BUFFERS_USAGE = "floatingBuffersUsage";
```

Comment:
floatingBuffersUsage -> inputFloatingBuffersUsage, also add the `input` prefix for following `exclusiveBuffersUsage`, then it is easy to distinguish from the metric name and keep consistent with other.